app/src/main/java/org/gdg/frisbee/android/arrow/ArrowTaggedActivity.java

```diff
@@ -120,7 +119,10 @@ public void onResult(AppStateManager.StateResult stateResult) {
 
                     @Override
                     public void success(final Directory directory, final retrofit.client.Response response) {
-                        String[] orgas = taggedOrganizers.split("\\|");
+                        if (orgas != null) {
```

Comment:
So. If you are in the tagged organizes viewing the list and hit Home button and then come back to the app the list just gets populated again having a duplicate of each item. It just keep growing and growing. That prevents the list from duplicating items once it has already been populated.